.github/workflows/docs.yml

```diff
@@ -8,23 +7,14 @@ on:
       - "[0-9]+.[0-9]+.[0-9]+"
   workflow_dispatch:
 
-env:
-  COLORTERM: 'yes'
-  TERM: 'xterm-256color'
-  PYTEST_ADDOPTS: '--color=yes'
-
 jobs:
   docs:
     runs-on: ubuntu-latest
     if: github.repository == 'kubevirt/kubevirt.core'
```

Comment:
Can you document why this is required? I assume because we can trigger against the ansible collection repo somehow?

Reply:
That could trigger if you have it in a private fork.